aspnetcore/security/authentication/accconfirm.md

```diff
@@ -1,38 +1,66 @@
 ---
-title: Account Confirmation and Password Recovery
+title: Account Confirmation and Password Recovery in ASP.NET Core
 author: rick-anderson
 description: Shows how to build an ASP.NET Core app with email confirmation and password reset.
 keywords: ASP.NET Core, password reset, email confirmation, security
 ms.author: riande
 manager: wpickett
-ms.date: 06/14/2017
+ms.date: 07/19/2017
 ms.topic: article
-ms.assetid: d794500b-86f7-4229-a237-e0dd00e2dc08
 ms.technology: aspnet
 ms.prod: asp.net-core
 uid: security/authentication/accconfirm
 ---
-# Account confirmation and password recovery
-
-<a name=security-authentication-account-confirmation></a>
+# Account confirmation and password recovery in ASP.NET Core
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
 This tutorial shows you how to build an ASP.NET Core app with email confirmation and password reset.
 
 ## Create a New ASP.NET Core Project
 
-The tutorial requires Visual Studio 2017 or higher.
+# [ASP.NET Core 2.x](#tab/aspnet20)
+
+The tutorial requires Visual Studio 2017 preview 2 or later.
+
+* In Visual Studio, create a New Web Application Project.
+* Select **ASP.NET Core 2.0**.
+* Select **Change Authentication** and set to **Individual User Accounts**.
+* Keep the default **Store user accounts in-app**.
+
+![New Project dialog showing "Individual User Accounts radio" selected](accconfirm/_static/2.png)
+
+# [ASP.NET Core 1.x](#tab/aspnet1x)
+
+The tutorial requires Visual Studio 2017 or later.
 
 * In Visual Studio, create a New Web Application Project.
 * Select **Change Authentication** and set to **Individual User Accounts**.
 
 ![New Project dialog showing "Individual User Accounts radio" selected](accconfirm/_static/indiv.png)
 
+---
+
+### CLI project creation for macOS and Linux
+
+If you're using the CLI or SQLite, run the following in a command window:
+
+```
```

Comment:
I recommend adding "console" as the type for this code fence. That makes it consistent with other docs and reinforces the fact that the CLI is being used.